src/pages/index.js

```diff
@@ -140,6 +201,10 @@ cardAddButton.addEventListener("click", () => {
   addFormValidator.resetValidation();
 });
 
+const deleteCardPopup = new PopupWithConfirmation("#delete-modal");
+
+deleteCardPopup.setEventListeners();
+
```

Comment:
Needs correction: When you click on the avatar image the avatar popup should open (the user avatar should be editable)